### ISSUE TITLE
IR Remote GZ-1002A-E3, Fuego ???

### ISSUE THREAD

**Issue by nemocaledonie** [esphome/issues#2723]
### The problem

Hello everyone,
I have a Galanz Air conditioner with a GZ-1002A-E3 remote control which according to the net would be recognized as Fuego, however it doesn't work.
I also tried with raw code which also doesn't work.

Doubting the functioning of my IR Mini shield, I tried with another library: IRremoteESP8266 via the Arduino IDE and used the dump (identical to that of ESPHome) and there it works (but except ESPHome and I do not know how integrate an external library to ESPHome).
In short if someone has a solution that would be nice, otherwise I will continue to use my remote control;)

### Which version of ESPHome has the issue?

2021.11.1

### What type of installation are you using?

Home Assistant Add-on

### Which version of Home Assistant has the issue?

2021.11.3

### What platform are you using?

ESP8266

### Board

Wemos Mini

### Component causing the issue

IR Shield Mini
